<launch>

    <arg name="arm" default="l"/>

    <!-- load planning -->
    <!--include file="$(find move_arm_tutorials2)/launch/ompl_planning.launch"/-->

    <!-- load move_arm -->
    <!--include file="$(find pr2_arm_navigation_config)/launch/pr2_planning_environment.launch" /-->
    <!--include file="$(find pr2_arm_navigation_actions)/launch/move_left_arm.launch"/-->
    <!--include file="$(find planning_environment)/launch/environment_server.launch"/-->

    <node pkg="pr2_arm_kinematics" type="pr2_arm_kinematics_node" name="pr2_left_arm_kinematics" output="screen">
      <param name="tip_name" value="l_gripper_spoon_frame" />
      <param name="root_name" value="torso_lift_link" />
    </node>


    <rosparam command="load" file="$(find hrl_feeding_task)/params/mpc_params_pr2.yaml" />
    <rosparam command="load" file="$(find hrl_feeding_task)/params/pr2_config_params_l.yaml" />
    <node name="arm_server" pkg="hrl_feeding_task" type="arm_reacher_server.py" args="-a $(arg arm)" output="screen"/>
    <!--Added by Hyder March 26 for testing...-->

    <!-- <node name="bowl_location_publisher" pkg="hrl_feeding_task" type="bowl_location_publisher.py" output="screen"/> -->

    <!--group ns="left" >

      <rosparam command="load" file="$(find hrl_anomaly_detection)/params/mpc_params_pr2.yaml" />
      <rosparam command="load" file="$(find hrl_anomaly_detection)/params/pr2_config_params_r.yaml" />

      <node name="arm_server" pkg="hrl_feeding_task" type="arm_reacher_server.py" args="-a $(arg arm)" output="screen"/>



    </group-->

</launch>
